buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-Any-Platform

```diff
@@ -56,6 +56,21 @@ timestamps {
         add_badges(downstreamBuilds)
         add_summary_badge(downstreamBuilds)
     }
+    try {
```

Comment:
Is this the best spot for this? I'm just wondering if `build_with_slack` would make more sense? Just wondering.